packages/cli/src/commands/setup/jobs/jobsHandler.js

```diff
@@ -0,0 +1,143 @@
+  const schema = await getDMMF({ datamodelPath: getPaths().api.dbSchema })
+
+  return schema.datamodel.models.map((model) => model.name)
+}
+
+const addModel = () => {
+  const schema = fs.readFileSync(getPaths().api.dbSchema, 'utf-8')
+
+  const schemaWithUser = schema + MODEL_SCHEMA
+
+  fs.writeFileSync(getPaths().api.dbSchema, schemaWithUser)
+}
+
+const tasks = ({ force }) => {
+  let skipSchema = false
+
+  return new Listr(
+    [
+      {
+        title: 'Creating job model...',
+        task: async (_ctx, task) => {
+          if ((await getModelNames()).includes('BackgroundJob')) {
+            skipSchema = true
+            task.skip('Model already exists, skipping creation')
+          } else {
+            addModel()
+          }
+        },
+      },
+      {
+        title: 'Migrating database...',
+        task: async (_ctx, task) => {
+          if (skipSchema) {
+            task.skip('Model already exists, skipping migration')
+          } else {
+            execa.sync(
+              'yarn rw prisma migrate dev',
+              ['--name', 'create-background-jobs'],
+              {
+                shell: true,
+                cwd: getPaths().base,
+                stdio: 'inherit',
+              },
+            )
+          }
+        },
+      },
+      {
+        title: 'Creating config file in api/src/lib...',
+        task: async () => {
+          const isTs = isTypeScriptProject()
+          const outputExtension = isTs ? 'ts' : 'js'
+          const outputPath = path.join(
+            getPaths().api.lib,
+            `jobs.${outputExtension}`,
+          )
+          let template = fs
+            .readFileSync(
+              path.resolve(__dirname, 'templates', 'jobs.ts.template'),
+            )
+            .toString()
+
+          if (!isTs) {
+            template = await transformTSToJS(outputPath, template)
+          }
+
+          writeFile(outputPath, template, {
+            overwriteExisting: force,
+          })
+        },
+      },
+      {
+        title: 'Creating jobs dir at api/src/jobs...',
+        task: () => {
+          try {
+            fs.mkdirSync(getPaths().api.jobs)
+          } catch (e) {
+            // ignore directory already existing
+            if (!e.message.match('file already exists')) {
+              throw new Error(e)
+            }
+          }
+          writeFile(path.join(getPaths().api.jobs, '.keep'), '', {
+            overwriteExisting: force,
+          })
+        },
+      },
+      {
+        title: 'One more thing...',
+        task: (_ctx, task) => {
+          task.title = `One more thing...
+          ${c.green('Background jobs configured!\n')}
+          ${'Generate jobs with:'} ${c.warning('yarn rw g job <name>')}
+          ${'Execute jobs with:'}  ${c.warning('yarn rw jobs work\n')}
+          ${'Check out the docs for more info:'}
+          ${chalk.hex('#e8e8e8')('https://docs.redwoodjs.com/docs/background-jobs')}
```

Comment:
With https://github.com/redwoodjs/redwood/pull/10939 this should now use `c.link`

```suggestion
          ${c.link('https://docs.redwoodjs.com/docs/background-jobs')}
```

Reply:
Hmm I don't love the success message and the commands both being green (using `tip`)...orange has the nicest contrast to me: 

<img width="441" alt="image" src="https://github.com/user-attachments/assets/c6fd6cd7-7a23-4771-8caf-58d7f3ad933a">

Tying the colors to log severity isn't always ideal, it's not a "bad" warning, I just want a color that looks nice and is easy to read. I don't want to never be able to use orange because we consider it a scary warning message. Orange just looks nice sometimes!

We've gotta do something about that `link` color for sure. I dunno where that standard came from (using #e8e8e8) because it's basically white if you have a dark terminal, and almost unreadable if you have a light one:

<img width="457" alt="image" src="https://github.com/user-attachments/assets/94393abf-0854-4313-be9b-70b8b7c321ba">

<img width="407" alt="image" src="https://github.com/user-attachments/assets/460a4835-b5b8-4277-91b4-72716944225d">